<?xml version="1.0"?>

<launch>

  	<node pkg="pcl_fusion" type="pcl_fusion_node" name="pcl_fusion_node" output="screen"/>
        <param name="/inputTopic1" value="/bumper_lidar/points" type="string" />
        <param name="/inputTopic2" value="/roof_lidar/points" type="string" />
        <param name="/outputTopic" value="/jeep/lidar_points" type="string" />


</launch>
